pillar/components/firewall.sls

```diff
@@ -2,43 +2,9 @@ firewall:
   data_public:
     services: []
     ports:
-      consul:
-        - 8600/tcp
-        - 8600/udp
-        - 8500/tcp
-        - 8301/tcp
-        - 8301/udp
-        - 8302/tcp
-        - 8302/udp
-        - 8300/tcp
-      dhclient:
-        - 68/udp
-      dhserver:
-        - 67/udp
       haproxy:
         - 443/tcp
-      hare:
-        - 8008/tcp
-      nfs:
-        - 2049/tcp
-        - 2049/udp
-        - 32803/tcp
-        - 892/tcp
-        - 875/tcp
-      s3:
```

Comment:
`9443/tcp` has to be part of s3 and not haproxy.

Reply:
@ypise addressed ! Thanks.